src/Employees/Models/EmployeeEntity.cs

```diff
@@ -30,4 +30,6 @@ public record EmployeeEntity
     public string? Address { get; set; }
     public string? ZipCode { get; set; }
     public string? City { get; set; }
+
+    public string? AllergyComment { get; set; }

```

Comment:
Om vi lager egne tabeller for Allergies og Preferences, gir det ikke mening å _ikke_ ha noe tilknyttet dette på selve Employees-tabellen? Vi har kyttet andre veien på alt annet

Hva er forskjellen på OtherAllergy og AllergyComment? De er begge fritekstfelt